src/ducks/SANCharts/ChartPage.js

```diff
@@ -26,7 +31,10 @@ const DEFAULT_STATE = {
   projectId: '16912',
   interval: getNewInterval(FROM, TO, '1d'),
   isAdvancedView: false,
-  enabledViewOnlySharing: true
+  enabledViewOnlySharing: true,
+  hideSettings: {
```

Comment:
Should be passed as a prop

Reply:
It is default props.

Reply:
You made it a part of the `DEFAULT_STATE` data structure. And also, why hiding it by default when it was the part of the new design when you was working on the PR?

Reply:
This new design with this block exists only in this page as part of widget. And it is hidden by default in other places. @DmitriVanGuard

Reply:
But will remove. It related by changes in chart widget